code/game/machinery/computer/timeclock_vr.dm

```diff
@@ -208,8 +208,9 @@
 /obj/machinery/computer/timeclock/proc/checkCardCooldown()
 	if(!card)
 		return FALSE
-	if((world.time - card.last_job_switch) < 15 MINUTES)
-		to_chat(usr, "You need to wait at least 15 minutes after last duty switch.")
+	var/time_left = 10 MINUTES - (world.time - card.last_job_switch)
+	if((time_left > 0)
```

Comment:
Missing parentheses here.

Reply:
I don't remember touching that...

Reply:
Aro sorta made an edit to the PR, yeah.

Reply:
It would be easier to just make a new Pr, @Arokha